pkgs/development/python-modules/bsddb3/default.nix

```diff
@@ -21,8 +21,16 @@ buildPythonPackage rec {
   build-system = [ setuptools ];
 
   buildInputs = [ pkgs.db ];
-
-  doCheck = pythonOlder "3.12"; # distutils usage
+  # See : https://github.com/NixOS/nixpkgs/pull/311198#discussion_r1599257522
+  # More details here : https://www.jcea.es/programacion/pybsddb.htm
+  broken = pythonAtLeast "3.9" || pythonOlder "3.5";
```

Comment:
`pythonOlder "3.5"` should be removed, since the link you provided specifically says that only this package of the two can support python <= 3.5